pkg/smart/parse.go

```diff
@@ -0,0 +1,329 @@
+package smart
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"regexp"
+	"strconv"
+	"strings"
+	"syscall"
+
+	"github.com/pkg/errors"
+)
+
+type ataSmartAttributes struct {
```

Comment:
Use `ATASMARTData`?

Reply:
The correct naming is `ataSMARTAttributes` or `ataSMARTData`